ansible/roles/common-cartridge/tasks/main.yml

```diff
@@ -8,14 +8,28 @@
   tags:
     - configmap
 
+- name: External Secret
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: external-secret.yml.j2
+    state: "{{ 'present' if 
+               WITH_BRANCH_MONGO_DB_MANAGEMENT is defined and WITH_BRANCH_MONGO_DB_MANAGEMENT|bool and

```

Comment:
You can shorten these expressions by writing WITH_BRANCH_MONGO_DB_MANAGEMENT | default(false)

Reply:
That's true. I used this expression to keep them consistent with the style that is used in the roles. I think mixing them could be a bit confusing.